Content.Server/Singularity/EntitySystems/EventHorizonSystem.cs

```diff
@@ -284,16 +284,23 @@ public int AttemptConsumeTiles(EntityUid hungry, IEnumerable<TileRef> tiles, Ent
     /// Checks whether an event horizon can consume a given tile.
     /// This is only possible if it can also consume all entities anchored to the tile.
     /// </summary>
-    public bool CanConsumeTile(EntityUid hungry, TileRef tile, MapGridComponent grid, EventHorizonComponent eventHorizon)
+    public bool CanConsumeTile(EntityUid hungry, TileRef tile, Entity<MapGridComponent> grid, EventHorizonComponent eventHorizon)
```

Comment:
```suggestion
    public bool CanConsumeTile(Entity<EventHorizonComponent> hungry, TileRef tile, Entity<MapGridComponent> grid)
```
might as well make it even more future proof